.github/workflows/build.yml

```diff
@@ -27,21 +24,40 @@ on:
         description: "A short note about this build (optional)"
         type: string
         required: false
+      will_export:
+        description: "Export build outputs"
+        type: boolean
+        required: true
+        default: false
 
 jobs:
+  health_check:
+    name: Health Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check inputs
+        run: |
+          echo "Workflow inputs are:"
+          echo "${{ toJSON(github.event.inputs) }}"
   template:
     if: inputs.dataset_name == 'template'
     uses: ./.github/workflows/template_build.yml
     secrets: inherit
     with:
-      do_export: ${{ inputs.do_export }}
       logging_level: ${{ inputs.logging_level }}
       build_note: ${{ inputs.build_note }}
+      will_export: ${{ inputs.will_export }}
```

Comment:
Are these doing anything? because below it seems that it's still just using the inputs anyways

Reply:
this line is only for the job to build the Temple dataset. in the jobs below,  the input is being referenced because the jobs are all independent

Reply:
Ohh sorry. Gotcha